features/ubuntu_pro.feature

```diff
@@ -27,8 +27,46 @@ Feature: Command behaviour when attached to an UA subscription
             fips-updates  +<fips> +<fips-s> +Uncertified security updates to FIPS modules
             livepatch     +yes      +enabled  +Canonical Livepatch service
             """
+        When I run `apt-cache policy` with sudo
+        Then stdout matches regexp:
+        """
+        https://esm.ubuntu.com/infra/ubuntu <release>-infra-updates/main amd64 Packages
```

Comment:
We might want to add a @then('apt-cache policy for {URL} has permission {perm_id}') to check for -32768 vs 500